package.json

```diff
@@ -168,6 +169,7 @@
     "react-native-sortable-listview": "git://github.com/EdgeApp/react-native-sortable-listview.git",
     "react-native-sound": "^0.11.0",
     "react-native-store-review": "^0.1.5",
+    "react-native-svg": "^12.1.0",
```

Comment:
Remove react native art

Reply:
Cannot remove yet until I replaced the loader on the Wallet List